packages/wallets/src/AppContent.scss

```
@@ -6,7 +6,7 @@
     width: 100%;
     align-self: stretch;
     background: var(--system-light-7-secondary-background, #f2f3f4);
-    height: calc(100vh - 8.4rem); // 100vh - (4.8rem header + 3.6rem footer)
+    min-height: calc(100vh - 8.4rem); // 100vh - (4.8rem header + 3.6rem footer)
```

Looks fine but definitely we can do something better.

---

```suggestion
    height: calc(100% - 8.4rem);
```
If you change it to this everything will work as before.